src/tests/testEvent.cc

```diff
@@ -66,20 +60,21 @@ testEvent::testCancel()
     CPPUNIT_ASSERT_EQUAL(0, event_to_cancel.calls);
 }
 
-/* submit two callbacks, and then dump the queue.
- */
+// submit two callbacks, and then dump the queue.
 void
 testEvent::testDump()
 {
     EventScheduler scheduler;
     CalledEvent event;
     CalledEvent event2;
-    CapturingStoreEntry * anEntry = new CapturingStoreEntry();
-    String expect =  "Last event to run: last event\n"
+    MemBuf expect;
+    expect.init();
+    expect.append("Last event to run: last event\n"
```

Comment:
We should not force developers to count characters in a long c-string with escape sequences!
* If possible, please MemBuf::vappendf("%s") instead of append().
* Otherwise, feed append() a c-string constant and strlen():
```C++
const auto expectedOutput = "Last event...
expect.append(expectedOutput, strlen(expectedOutput));
```

Reply:
Done in 06101f8